dbt_semantic_interfaces/protocols/query_interface.py

```diff
@@ -42,6 +61,8 @@ def create(
         self,
         time_dimension_name: str,
         time_granularity_name: str,
+        descending: Optional[bool] = None,
+        date_part_name: Optional[str] = None,
```

Comment:
Should these also be added to the QueryInterfaceDimensionFactory.create() method?

Reply:
Oops, I somehow forgot about this between writing it and submitting the review. If this was an oversight please fix before merge!